src/Resources/public/Admin.js

```diff
@@ -226,6 +226,10 @@ var Admin = {
                     .replaceWith(html);
             },
             error: function(xhr, statusText, errorThrown) {
+                if (xhr.status === 400){
```

Comment:
Why only for 400 ?

Do you know if we have to JSON.parse for every case ?
If not, I think a comment could help to understand this check.

Reply:
Yes indeed JSON.parse is also needed for 405, 404, 403, i will update this in my pull request.

Reply:
Is there some case where you don't need `JSON.parse` ?

If there are not, we can just write
```
error: function(xhr, statusText, errorThrown) {
    return JSON.parse(xhr.responseText);
}
```

If there are, how did you decide (between using JSON.parse) for them and for 400, 403, 404, 405 ?

Reply:
No because if is a 500 throwed you don't see it, it's html not Json

Reply:
ahhh you are right, in production it's not a problem… i speak in case of dev